bluefin-internal/src/Bluefin/Internal/Examples.hs

```diff
@@ -1021,3 +1021,23 @@ rethrowIOExample = runEff $ \io -> do
   effIO io $ putStrLn $ case r of
     Left e -> "Caught IOException:\n" ++ show e
     Right contents -> contents
+
+data LocalReader r e = LocalReader
+  { askLR :: forall es. Eff (es :& e) r
+  , localLR :: forall es a. (r -> r) -> Eff (es :& e) a -> Eff (es :& e) a
```

Comment:
I'm _very_ surprised that `e` can appear in negative position here, yet not prevent `mapHandle` from being defined. Is my intuition completely wrong?

Reply:
I also didn't think it was possible until just now. This type is equivalent to `forall es a. e :> es => (r -> r) -> Eff es a -> Eff es a` which has a more obvious polarity.

Reply:
Wow, my intuition really needs upgrading.